portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -65,34 +65,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Comment objects added to comments ArrayList will be displayed
     ArrayList<Comment> comments = new ArrayList<Comment>();
-    // count variable is the total number of comments linked with the current img
-    int count = 0;
-    // Iterating over all comments with various img tags
     for (Entity entity : results.asIterable()) {
-      String img = (String) entity.getProperty("img");
-
-      // Only add the comment if it's linked with the current img
-      if (img.equals(imgRequest)) {
-        long id = entity.getKey().getId();  
-        String name = (String) entity.getProperty("name");
-        String comment = (String) entity.getProperty("comment");
-        long timestamp = (long) entity.getProperty("timestamp");
+      long id = entity.getKey().getId();  
```

Comment:
Code change below is so much cleaner! Nice

Reply:
Agree!

Reply:
Thanks!